packages/htmlbars-compiler/lib/compiler.js

```diff
@@ -1,6 +1,7 @@
 /*jshint evil:true*/
 import { preprocess } from "../htmlbars-syntax/parser";
 import TemplateCompiler from "./template-compiler";
+import { wrap } from "../htmlbars-runtime/hooks";
```

Comment:
Nice. If `utils` is imported into both `runtime` and `compiler`, that's the right place for this for sure.